ada/ast.py

```diff
@@ -2995,6 +2995,16 @@ def wrap_public_reference():
             default_val=Entity
         )
 
+    @langkit_property(public=True, return_type=Bool)
```

Comment:
Use abstract_runtime_check to have this behavior automatically (check other properties in ast.py)

Reply:
Should I also modify the propertyError message for something like `should not happen` instead?

Reply:
Even better, as mentioned live by @Roldak , just use `pass` here.